scheds/rust/scx_rustland/src/main.rs

```diff
@@ -290,16 +290,16 @@ impl<'a> Scheduler<'a> {
         let init_page_faults: u64 = 0;
 
         // Low-level BPF connector.
-        let nr_online_cpus = topo_map.nr_cpus_online();
+        let nr_cpus = topo_map.nr_cpus_possible();
```

Comment:
Note that possible cpumask can have holes and thus the number of possible CPUs may deviate from the last CPU ID + 1. In the kernel, the difference is between `num_possible_cpus` and `nr_cpu_ids`. They are usually the same but not always.

@Byte-Lab Skimming the code, it looks like while the topo code would report the correct `span` and `nr_cpus_possible`, reporting the two combo is likely to mislead users as it's easy for people to assume that the possible cpu IDs are consecutive. It'd probably be better to report `nr_cpu_ids` and document the fact that it may deviate from the number of possible CPUs.

Reply:
Good point. We definitely need `nr_cpu_ids` in this case. I'm also wondering if I should just use `MAX_CPUS` to create all the DSQs, without worrying about potential holes or offline CPUs, in this way we should even be able to handle CPU hotplugging events.

Reply:
`nr_cpu_ids` should do the job. CPU hotplugging can ever only happen among the possible CPUs, or you can walk the possible cpumask and create a DSQ for each possible CPU too.